### DISCUSSION TITLE
Updating some of the legacy Usage pages

### DISCUSSION THREAD

**Discussion by whugemann** [ImageMagick/ImageMagick#4541] [Contribute]
The subpages 'Usage under Windows' and 'Lens correction' were mostly written by me. I sent the HTML code to Anthony an he formatted and uploaded it. 

Next year I will probably find the time to update my pages, i.e. use IM version 7 and add some better solutions for organizing scripts in a computer network environment. Also, I am now much more versed in wirting batch files and will probably have to rewrite some of the text.

Is there any interest in me doing this and how would that practically work?

**Comment by urban-warrior** [comment]
Yes of course, we are interested in your contributions to the legacy pages.  They are quite valuable and should be enhanced as well as  updated to version 7 of ImageMagick.  We added Anthony's (and your's) usage pages to a repo @ https://github.com/ImageMagick/Usage.  You can create pull requests for any mods if you are only going to update a few sections.  If you want to make a serious effort to improve most or all pages, we could provide developer access.

**Reply by whugemann** [reply]
OK, I understand this all works via git now. I suggest that I will start with the 'lens' section next year, which should be a manageable task for me. If this works out, I will start to rewrite the Windows section. And possibly, I will try to transform some of Anthony's pages to version 7, although these are large shoes to walk in ... Have you checked with Anthony how we will have to credit his work?

**Comment by urban-warrior** [comment]
We notified Anthony that we were hosting his web pages in Gitlab and we did not receive a non-object.  You are welcome to contact him directly if you need clarification.
